### ISSUE TITLE
Is it possible to run multiple scripts on the same prometheus job/endpoint

### ISSUE THREAD

**Issue by enidvx** [ricoberger/script_exporter#100]
Hi, 
I have this situation where I want to add a prometheus job to scrape an endpoint and on the server `script_exporter` would execute let's say 3 scripts. Is it possible to scrape them on the same job like this?

```
  - job_name: "oracle-scripts"
    scrape_interval: "1h"
    scrape_timeout: "1m"
    scheme: "http"
    metrics_path: "/probe"
    params:
      script:
        - "check_script1"
        - "check_script2"
        - "check_script3"
    file_sd_configs:
      - files:
          - "/etc/prometheus/file_sd/targets.yml"
```
If I run the above I get the metrics only of the first script `check_script1` meaning that the other 2 scripts are not executed.
The endpoint scraped from above would be:
`http://target_server:9469/probe?script=check_script1&script=check_script2&script=check_script3`

Thanks,
Enid

**Comment by ricoberger** [comment]
Hi @enidvx, no this is currently not possible, but I think this can be implemented in a nice way. I will have a look at this later today.

**Comment by ricoberger** [comment]
Hi @enidvx, the feature should be available in [v2.16.0](https://github.com/ricoberger/script_exporter/releases/tag/v2.16.0).
